apps/CardinalBanditsPureExploration/myApp.yaml

```diff
@@ -42,22 +42,28 @@ initExp:
           default: 0.5
           optional: true
 
+        pT:
```

Comment:
Why is this required? The YAML file is for algorithm inputs/outputs. `pT` is internal to the algorithm.

Reply:
I could not get butler to work/save parameters unless the variable in the yaml. I am not sure of the 'correct' way to get butler working.

Reply:
Hm... that's not how `butler` is supposed to work. It looks like this should work if this is removed beacuse `pT` is initialized. If it doesn't, could you give a traceback?